playbooks/terraform-provider-openstack-acceptance-test-opentelekomcloud/run.yaml

```diff
@@ -0,0 +1,45 @@
+- hosts: all
+  become: yes
+  roles:
+    - config-golang
+    - role: export-cloud-openrc
+      vars:
+        cloud_name: 'opentelekomcloud'
+  tasks:
+    - name: Run acceptance tests with terraform-provider-openstack against opentelekom public cloud
+      shell:
+        cmd: |
+          apt-get install python-pip -y
+          pip install -U python-openstackclient
+
+          set -o pipefail
+          set -x
+          export OS_SHARE_NETWORK_ID="foobar"
+          export OS_FLAVOR_ID_RESIZE='computev2-1'
+          export OS_FLAVOR_ID='computev1-1'
+          export OS_POOL_NAME="admin_external_net"
+          export OS_EXTGW_ID=`openstack network list -f value |grep admin_external_net | awk -F ' ' '{print $1}'`
+          export OS_IMAGE_NAME=cirros-0.4.0-x86_64-disk
+          export OS_IMAGE_ID=f0ee30ed-1fcd-4d7c-8326-2178fe9dfe5b
```

Comment:
Why do not use openstackclient command to avoid hard code uuid, like you do in huaweicloud yaml